.github/workflows/node_tests.yml

```diff
@@ -50,6 +56,28 @@ jobs:
           npm ci
         working-directory: ${{ inputs.working_directory }}
 
+      - name: Collect packages licenses
+        if: inputs.check_packages_licenses
+        run: |
+          echo '{"name":"", "license":""}' > fmt.json
+          npx -y license-checker@25.0.1 --csv --production --excludePrivatePackages --customPath fmt.json | cut -d, -f2,3 | tail -n +2 > licenses.csv
+          cat licenses.csv
+        working-directory: ${{ inputs.working_directory }}
+
+      - name: Check packages licenses
+        if: inputs.check_packages_licenses
+        id: license_check_report
+        continue-on-error: true
+        uses: pilosus/action-pip-license-checker@v0.6.2
+        with:
+          requirements: ${{ inputs.requirements_path }}
+          external: ${{ inputs.working_directory }}/licenses.csv
+          external-format: csv
+          exclude: uWSGI.*|lunardate.*|.*QuokkaClient.*
```

Comment:
You should not need the .* here because you are trimming the version on the npx command